Automattic/Simplecons
# Add help icon

# Fix
Add a help icon (16px). See below for illustration.

<img width="147" alt="Screenshot 2021-05-05 at 12 16 27" src="https://user-images.githubusercontent.com/36532468/117133114-b8fc2300-ad9b-11eb-89ff-621ff4864b6a.png">


### To Test
Verify the icon appears as intended on a 16px grid.
Verify the SVG code is consistent with other icons.

Also updating .gitignore since some mac system files were not being ignored.
